pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -158,6 +168,7 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&CiliumNodeList{},
 		&CiliumIdentity{},
 		&CiliumIdentityList{},
```

Comment:
[1] From our offline discussion. The reason why the was failing was because the missing types here:
```golang
&CiliumLocalRedirectPolicy{},
&CiliumLocalRedirectPolicyList{},
&CiliumClusterwideLocalRedirectPolicy{},
&CiliumClusterwideLocalRedirectPolicyList{},
```